tests/main/canonical-livepatch-14.04/task.yaml

```diff
@@ -0,0 +1,36 @@
+summary: Ensure the canonical-livepatch snap works on 14.04
+
+description: |
+    This is a specific 14.04 test that ensure live-patch work
+    with a snapd that re-execs on 14.04. We don't regularly
+    update the deb on 14.04 anymore so we need to be sure that
+    re-exec is enough to run live-patch.
+
+# livepatch works only on amd64 systems
+systems: [ubuntu-14.04-64]
+
+execute: |
+    echo "Keep a copy of the core snap with our latest snapd"
+    cp /var/lib/snapd/snaps/core_*.snap .
+
+    echo "And then install the snapd package from the trusty archvie"
+    #shellcheck source=tests/lib/pkgdb.sh
+    . "$TESTSLIB"/pkgdb.sh
+    distro_purge_package snapd
```

Comment:
What undoes this step?

Reply:
Thanks, good catch! There is a restore missing here. I added it now.